File: programacion/java/1_unidad/000ejercicos_examen/MayorDeTres.java
import java.util.Scanner;
public class MayorDeTres{
	public static void main(String[] args){

		Scanner teclado = new Scanner(System.in);

		System.out.println("COMPARAR 3 NÚMEROS:");
			
		System.out.print("Introduce el primer numero: ");

		double num1 = Double.parseDouble(teclado.nextLine());

		System.out.print("Introduce el segundo numero: ");

		double num2 = Double.parseDouble(teclado.nextLine());

		System.out.print("Introduce el tercer numero: ");

		double num3 = Double.parseDouble(teclado.nextLine());



		if (num1 > num2 && num1 > num3) {
			System.out.println("Solo el primer número (" + num1 + ") es el mayor.");
		} else if (num2 > num1 && num2 > num3) {
			System.out.println("Solo el segundo número (" + num2 + ") es el mayor.");
		} else if (num3 > num1 && num3 > num2) {
			System.out.println("Solo el tercer número (" + num3 + ") es el mayor.");
		} else if (num1 == num2 && num1 > num3) {
			System.out.println("El primer y segundo número (" + num1 + ") son los mayores e iguales.");
		} else if (num2 == num3 && num2 > num1) {
			System.out.println("El segundo y tercer número (" + num2 + ") son los mayores e iguales.");
		} else if (num1 == num3 && num1 > num2) {
			System.out.println("El primer y tercer número (" + num1 + ") son los mayores e iguales.");
		} else if (num1 == num2 && num1 < num3) {
			System.out.println("El tercer número (" + num3 + ") es el mayor, los otros dos son iguales.");
		} else if (num2 == num3 && num2 < num1) {
			System.out.println("El primer número (" + num1 + ") es el mayor, los otros dos son iguales.");
		} else if (num1 == num3 && num1 < num2) {
			System.out.println("El segundo número (" + num2 + ") es el mayor, los otros dos son iguales.");
		} else {
			System.out.println("Los tres números son iguales.");
		}

		teclado.close();
	}
}

File: programacion/python/unidad_1/000ejercicios_examen/MayorDeTres.py
num1 = int(input("Introduce el primer número: "))
num2 = int(input("Introduce el segundo número: "))
num3 = int(input("Introduce el tercer número: "))

if num1 > num2 and num1 > num3:
    print(f"El número {num1} es el mayor de los 3")
elif num2 > num1 and num2 > num3:
    print(f"El número {num2} es el mayor de los 3")
elif num3 > num1 and num3 > num2:
    print(f"El número {num3} es el mayor de los 3")
elif num1 == num2 and num1 > num3:
    print(f"El primer y segundo número ({num1}) son los mayores e iguales")
elif num2 == num3 and num2 > num1:
    print(f"El segundo y tercer número ({num2}) son los mayores e iguales")
elif num1 == num3 and num1 > num2:
    print(f"El primer y tercer número ({num1}) son los mayores e iguales")
elif num1 == num2 and num1 < num3:
    print(f"El tercer número ({num3}) es el mayor, los otros dos son iguales")
elif num2 == num3 and num2 < num1:
    print(f"El primer número ({num1}) es el mayor, los otros dos son iguales")
elif num1 == num3 and num1 < num2:
    print(f"El segundo número ({num2}) es el mayor, los otros dos son iguales")
else:
    print("Los tres números son iguales")